docs/01-about/index.md

```diff
@@ -1,37 +1,58 @@
+::: info
+Nordcraft is for designers **and** developers, whether you're working on your next big side project, or as part of a professional product team.
+:::
+
+## Unifying design and development
+
+Nordcraft combines a visual editor for interface design with a powerful technical foundation built on modern web standards, accelerating development of your websites and web applications without sacrificing quality or control.
+
+Key features include:
+
+- Visual editing of UI components familiar to designers working in Figma
 - Component-based architecture familiar to React, Vue, and Svelte developers
-- Visual development for implementing interactive behavior
-- Branch-based development for team collaboration
+- Visual development environment for implementing interactive behavior and application logic
+- Branch-based environment for parallel feature development and team collaboration
 
 ## Technical foundation
-Built on a lightweight 24kb runtime with a reactive architecture, Nordcraft supports both server-side and client-side rendering. The system prioritizes performance while providing developers with the tools they need for modern web applications.
+
+The Nordcraft Web Development Engine is a lightweight 24kb runtime with a reactive architecture, supporting both server-side and client-side rendering. Nordcraft prioritizes the performance of your applications while providing you with all the tools you need for modern web application development.
 
 Core technical features:
+
 - Signals-based reactive programming model
 - Full server-side rendering capabilities
 - Minimal DOM output for optimal performance
 - Animation engine for creating dynamic interfaces
 - Integrated version control with preview environments
+- Add your own custom code to your Nordcraft apps
 
-## Complete ecosystem
-Nordcraft provides everything needed to build and deploy web applications:
-- API integration for connecting to any backend service
+## The Nordcraft ecosystem
+
+Nordcraft provides everything you need to build, test, and deploy modern, performant web applications:
+
+- Full API integration for connecting to any backend service
 - Package system for sharing and reusing functionality
-- Templates for rapid application development and design systems
+- Pre-built templates for rapid application development and design systems
 - Asset management for handling media
 - SEO and meta tag configuration tools
-- Deployment to cloud or self-hosted infrastructure as Nordcraft is open-source
+- Live branch preview environments
+- Deploy directly to Nordcraft's infrastructure or self-host your applications
+
+The Nordcraft Web Development Engine gives you the freedom to fully customize and extend the system to meet your specific needs without limitations.
+
+## The Nordcraft community
+
+At Nordcraft, we're proud of our growing community where designers and developers are sharing their exciting projects built with Nordcraft.
```

Comment:
Should we mention how many (3,000+) members have joined our Discord server?

Reply:
YES